ember-model.js

```diff
@@ -340,29 +340,29 @@ Ember.ManyArray = Ember.RecordArray.extend({
   },
 
   replaceContent: function(index, removed, added) {
-    added = Ember.EnumerableUtils.map(added, function(record) {
+    added = added.map(function(record) {
       return record._reference;
     }, this);
 
     this._super(index, removed, added);
   },
 
-  _contentWillChange: function() {
+  _contentDidChange: function() {
```

Comment:
Heh, `package/tests` I had the same thought awhile ago. I'll dig into the tests a bit now and confirm there's coverage here.